extensions/jitsi/in-div.ejs

```diff
@@ -145,14 +155,27 @@
   height: 95%;
 }
 
+#jitsi-others-btn {
+  display: none;
+}
+
 </style>
 <div class="jitsi-container jitsi-container--inactive" id='jitsi-container'>
-  <div class='jitsi-remote-container' id='jitsi-remote-container'>
-
-  </div>
   <div class='jitsi-local-container' id='jitsi-local-container'>
     <div class='jitsi-user-container' id='jitsi-local'></div>
   </div>
+  <div class='jitsi-remote-container' id='jitsi-remote-container'>
+  </div>
+  <div>
+    <button id="jitsi-others-btn"></button>
+  </div>
 </div>
 
-<div id='jitsi-fullscreen-overlay'></div>
\ No newline at end of file
+<div id='jitsi-fullscreen-overlay'></div>
+
+<div id="jitsi-modal">
```

Comment:
Not sure what `#jitsi-modal` is for. Others LGTM.

Reply:
[TODO]
In the future, it will provide user to view all participants info in the meeting.